armi/physics/neutronics/crossSectionSettings.py

```diff
@@ -57,6 +58,8 @@
 CONF_COMPONENT_AVERAGING = "averageByComponent"
 CONF_XS_MAX_ATOM_NUMBER = "xsMaxAtomNumber"
 CONF_MIN_DRIVER_DENSITY = "minDriverDensity"
+CONF_PARTIALLY_HETEROGENEOUS = "partiallyHeterogeneous"
```

Comment:
I personally think the name of this setting should be more descriptive like `ductHeterogeneous`.